Shared/mods/deathmatch/logic/lua/CLuaFunctionParser.h

```diff
@@ -201,6 +203,10 @@ struct CLuaFunctionParserBase
         if constexpr (std::is_same_v<T, CLuaArgument>)
             return iArgument != LUA_TNONE;
 
+        // all color classes are read as 1 number, not 3 different.
+        if constexpr (std::is_base_of_v<SColor, T>)
```

Comment:
Should be is_same_v

Reply:
I thought we want to be able to read every derived class of SColor?

Reply:
See comment here https://github.com/multitheftauto/mtasa-blue/pull/1465/files#diff-eedaee2413b2da265460ed09b1abe646R206-R210